README.md

```diff
@@ -31,3 +31,15 @@
 - [Traffic Light](./examples/traffic-light)
 - [Sokoban](./examples/sokoban) -- A type of transport puzzle, in which the player pushes boxes or crates around in a warehouse, trying to get them to storage locations.
 - [Drag-n-Drop](./examples/drag-n-drop) -- A simple drag-n-drop reorderable list.
+
+## Developing
+
+### Requirements
+
+* Latest stable version of [Node](https://github.com/creationix/nvm)
+* Latest stable version of [Yarn](https://yarnpkg.com)
+
+### For generating changelog (for those with NPM publishing rights)
+
+* [Ruby >= v2.1.0](https://www.ruby-lang.org/en/downloads/)
```

Comment:
`gem install rake` is also required on Debian based systems such as Ubuntu and Linux Mint.

Reply:
export `CHANGELOG_GITHUB_TOKEN`

Reply:
Yeah these would be good to add 👍